src/plugins/imagecapturer/CImageCapturer.cpp

```diff
@@ -114,6 +119,10 @@ namespace semrec {
       try {
 	if(sensor_msgs::image_encodings::isColor(m_imgReceived.encoding)) {
 	  cv_ptr = cv_bridge::toCvCopy(m_imgReceived, sensor_msgs::image_encodings::BGR8);
+  } else if(isDepthImage(m_imgReceived.encoding)) {
```

Comment:
Thanks for the pull request. Looks good. If you fix the indentation, I'll merge it in.